docs/features/logforwarding/outputs/syslog-forwarding.adoc

```diff
@@ -0,0 +1,159 @@
+  The `payloadKey` must be a single field path encased in single curly brackets `{}`.
+  Field paths must only contain alphanumeric and underscores. Any field with other characters must be quoted.
+  If left empty, Syslog will use the whole message as the payload key.
+  Examples:
+
+  {.bar}
+  {.foo.bar.baz}
+  {.foo.bar."baz/with/slashes"}
+
+. `appName`: is APP-NAME part of the syslog-msg header.
+  `appName` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 48
+  This supports template syntax to allow dynamic per-event values.
+  The `appName` can be a combination of static and dynamic values consisting of field paths followed by `||` followed by another field path or a static value.
+  A dynamic value is encased in single curly brackets `{}` and MUST end with a static fallback value separated with `||`.
+  Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes.
+  Examples:
+
+  foo-{.bar||"none"}
+  {.foo||.bar||"missing"}
+  foo.{.bar.baz||.qux.quux.corge||.grault||"nil"}-waldo.fred{.plugh||"none"}
+
+.  `procId`: is `PROCID` part of the syslog-msg header.
+   `procId` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 128
+   This supports template syntax to allow dynamic per-event values.
+   The `procId` can be a combination of static and dynamic values consisting of field paths followed by `||` followed by another field path or a static value.
+   A dynamic value is encased in single curly brackets `{}` and MUST end with a static fallback value separated with `||`.
+   Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes. Example:
+
+   foo-{.bar||"none"}
+   {.foo||.bar||"missing"}
+   foo.{.bar.baz||.qux.quux.corge||.grault||"nil"}-waldo.fred{.plugh||"none"}
+
+.  `msgId`: is `MSGID` part of the syslog-msg header. This supports template syntax to allow dynamic per-event values.
+MsgId needs to be specified if using `rfc5424`.  The maximum length of the final values is truncated to 32
+The MsgId can be a combination of static and dynamic values consisting of field paths followed by `||` followed by another field path or a static value.
+A dynamic value is encased in single curly brackets `{}` and MUST end with a static fallback value separated with `||`.
+Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes.
+Example:
+
+    foo-{.bar||"none"}
+    {.foo||.bar||"missing"}
+    foo.{.bar.baz||.qux.quux.corge||.grault||"nil"}-waldo.fred{.plugh||"none"}
+
+. `enrichment`: `None` or `KubernetesMinimal`.
+`None` add no additional enrichment to the record.
+`KubernetesMinimal` adds `namespace_name`, `pod_name`, and `collector_name` to the beginning of the message
```

Comment:
should be container_name not collector_name

Reply:
copy/past error from here https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L1219

Reply:
fixed